index.js

```diff
@@ -94,9 +94,13 @@ module.exports = (context, options) => {
 
     // Cherry-pick lodash modules for smaller bundles
     if (options.lodash) {
+        const defaultLodashOptionsIds = ['lodash/fp'];
+
         config.plugins.push([
-            require.resolve('babel-plugin-lodash'),
-            options.lodash === true ? {} : options.lodash,
+            [require.resolve('babel-plugin-lodash')],
+            Object.assign({},
+                options.lodash,
+                { id: options.lodash.id ? defaultLodashOptionsIds.concat(options.lodash.id) : defaultLodashOptionsIds }),
```

Comment:
We should document the defaults in the README and perhaps and example adding recompose?

Reply:
I agree, i will add the information with an example in the README.

Reply:
+1